inventory_management_system_api/models/catalogue_item_property_template.py

```diff
@@ -0,0 +1,27 @@
+"""
+Module for defining the database models for representing a Catalogue item property template
+"""
+
+from typing import Optional
+
+from pydantic import BaseModel, ConfigDict, Field
+
+from inventory_management_system_api.models.catalogue_category import AllowedValues
+from inventory_management_system_api.models.custom_object_id_data_types import StringObjectIdField
+
+# pylint: disable=duplicate-code
+
+
+class CatalogueItemPropertyTemplateOut(BaseModel):
+    """
+    Model representing a catalogue item property template.
+    """
+
+    id: StringObjectIdField = Field(alias="_id")
```

Comment:
I see on the front end you just inherit and add the id. Cant you do the same with the model and schema here? The schema in particular also has some validation that would be needed if doing a create endpoint. Of course Viktor's PR will be adding IDs as well so perhaps its even simpler.